api/managementpb/backup/artifacts.proto

```diff
@@ -57,4 +66,11 @@ service Artifacts {
       body: "*"
     };
   }
+  // DeleteArtifact deletes specified artifact.
+  rpc DeleteArtifact(DeleteArtifactRequest) returns (DeleteArtifactResponse) {
+    option (google.api.http) = {
+      post: "/v1/management/backup/Artifacts/Remove"
```

Comment:
Why `Remove` if gRPC method is named `DeleteArtifact`?

Reply:
fixed, thanks